core/dia/endscreen/dia_endscreen.hpp

```diff
@@ -46,6 +46,7 @@ class DIA_ENDSCREEN {
 		w = 0.13125 * safezoneW;
 		h = 0.350035 * safezoneH;
 		colorText[] = {-1,-1,-1,-1};
+		size = "(((((safezoneW / safezoneH) min 1.2) / 1.2) / 25) * 0.75)";
```

Comment:
Where does the safezoneW and safezoneH variables get initiated? It looks like the same "size" variable is declared multiple times, could this be done once and referenced in all of the classes?

Reply:
safezone variables are loaded as soon as the graphics settings change/ on game load. They are automatically calculated by the engine depending on your aspect ratio/size of display.

Reply:
> safezone variables are loaded as soon as the graphics settings change/ on game load. They are automatically calculated by the engine depending on your aspect ratio/size of display.

OK, so could this size calculation not be put into a function rather than repeating the exact same code multiple times?

Reply:
You couldnt call a function in it, but you could make a macro for them

Reply:
OK sounds good.

Reply:
refer to last commit

Reply:
Cool.